README.md

```diff
@@ -268,6 +269,14 @@ The generation of a random filename can be overridden by sending a header called
 curl -F "file=@x.txt" -H "filename: <file_name>" "<server_address>"
 ```
 
+#### No extension when using `random_url`
+
+Rustypaste adds the filename's extension to the url. This can be changed by setting `no_extension` to `true`:
+
+```toml
+random_url = { type = "alphanumeric", length = 8, no_extension = true }
+```
+
```

Comment:
This feels too specific in the documentation... should we maybe remove it?

Also, we don't have any other config examples in README.md - I think the comment in the `config.toml` would help with demonstrating this already. It's quite an simple option anyways :D

Reply:
Hmm, I really don't know.

> the comment in the config.toml would help with demonstrating this already

That would require people to look for info in the `config.toml`. Personally I would, but I cannot deduce the behavior of others from this.

Do you suggest removing everything, but only lines 276-278?

Reply:
I suggest just removing that example entirely.

At some point we really need a dedicated website which we can host multiple short pages where we can do a tips & tricks section like this: https://git-cliff.org/docs/tips-and-tricks

Reply:
ok, I removed it again.

> we really need a dedicated website which we can host multiple short pages where we can do a tips & tricks section like this: https://git-cliff.org/docs/tips-and-tricks

not a bad idea.